app/views/children/index.html.erb

```diff
@@ -30,72 +30,62 @@
             <h3 class="card-title">Filters</h3>
           </div>
           <div class="card-body">
-            <%= form_tag(children_path, method: :get) do |f| %>
+            <%= form_for_filterrific @filterrific, remote: true do |f| %>
               <div class="row">
-                <% if @family.present? %>
-                  <div class="form-group col-lg-4 col-md-4 col-sm-6 col-xs-12">
-                    <%= label_tag "by Guardian Name", "Guardian Name" %>
-                    <%= collection_select(:filters, :from_family, @family || {}, :id, :guardian_display_name, {include_blank: true, selected: @selected_family}, class: "form-control") %>
-                  </div>
-                <% end %>
-                <% if @children.present? %>
-                  <div class="form-group col-lg-4 col-md-4 col-sm-6 col-xs-12">
-                    <%= label_tag "by Children First Name", "Child's First Name" %>
-                    <%= collection_select(:filters, :from_children, @children || {}, :id, :first_name, {include_blank: true, selected: @selected_children_first_name}, class: "form-control") %>
-                  </div>
-                <% end %>
+                <div class="col-2">
```

Comment:
```suggestion
                <div class="col-3">
```
I noticed when I tested it locally the labels were a bit squished (maybe due to my laptop?)
With `col-2`
<img width="1363" alt="Screen Shot 2020-06-19 at 10 10 21 AM" src="https://user-images.githubusercontent.com/11335191/85148064-50549800-b215-11ea-9287-dfa6d9d4dea0.png">

With `col-3`
<img width="1378" alt="Screen Shot 2020-06-19 at 10 10 35 AM" src="https://user-images.githubusercontent.com/11335191/85148083-55b1e280-b215-11ea-90e1-dd9f7dceca9d.png">

Reply:
I'll make the change! I'm on a 38 monitor and I imagine some of the diaper banks are not on giant monitors too!